proto/xudrpc.proto

```diff
@@ -405,6 +413,30 @@ message ListPeersResponse {
   repeated Peer peers = 1 [json_name = "peers"];
 }
 
+message ListOrderHistoryRequest {
+  string order_id = 1 [json_name = "order_id"]; 
+}
+
+message ListOrderHistoryResponse {
+  repeated Trade trades = 1 [json_name = "trades"];
+  repeated SwapSuccess swaps = 2 [json_name = "swaps"];
+  OrderStatus status = 3 [json_name = "status"];
+
+  enum OrderStatus {
+    FILLED  = 0;
+    PARTIALLYFILLED = 1;
+    OPEN = 2;
+    CANCELLED = 3;
+  };
+}
+
+message Trade {
```

Comment:
I think it would be nice to have the price in this message as well, so that someone can quickly see the price of the trade without having to look up the orders.

Reply:
I messaged you privately about this.